website/src/_includes/section-menu/contribute.html

```diff
@@ -12,20 +12,33 @@
 
 <li><span class="section-nav-list-main-title">Contribute</span></li>
 <li><a href="{{ site.baseurl }}/contribute/">Get started contributing</a></li>
-<li><a href="https://cwiki.apache.org/confluence/display/BEAM/Using+IntelliJ+IDE">Using IntelliJ IDE</a></li>
+<li><a href="https://cwiki.apache.org/confluence/display/BEAM/Contribution+FAQ">FAQ</a></li>
 <li>
   <span class="section-nav-list-title">Technical Docs</span>
 
   <ul class="section-nav-list">
+    <li><a href="{{ site.baseurl }}/roadmap/">Roadmap</a></li>
     <li><a href="{{ site.baseurl }}/contribute/testing/">Testing guide</a></li>
-    <ul>
-      <li><a href="{{ site.baseurl }}/contribute/precommit-triage-guide/">Pre-commit Slowness Triage Guide</a></li>
-    </ul>
+    <li><a href="{{ site.baseurl }}/contribute/precommit-triage-guide/">Pre-commit Slowness Triage</a></li>
     <li><a href="{{ site.baseurl }}/contribute/ptransform-style-guide/">PTransform style guide</a></li>
     <li><a href="{{ site.baseurl }}/contribute/runner-guide/">Runner authoring guide</a></li>
     <li><a href="{{ site.baseurl }}/contribute/docker-images/">Docker images</a></li>
     <li><a href="{{ site.baseurl }}/contribute/design-documents/">Design documents</a></li>
     <li><a href="{{ site.baseurl }}/contribute/dependencies/">Dependencies guide</a></li>
+    <li><a href="{{ site.baseurl }}/contribute/feature-branches/">Feature branches</a></li>
+  </ul>
+</li>
+<li>
+  <span class="section-nav-list-title">Tips</span>
+  <ul class="section-nav-list">
+    <li><a href="https://cwiki.apache.org/confluence/display/BEAM/Using+IntelliJ+IDE">Using IntelliJ IDE</a></li>
+    <li><a href="{{ site.baseurl }}/contribute/java-tips/">Java development tips</a></li>
```

Comment:
How about moving this tips section to the wiki is well? It would be easier to maintain / update there?

Reply:
Done. Moved all the tips to the wiki. There are pros + cons; I like the review process having these in github but it easier to edit as wiki pages.

Reply:
Yup. IMO ease of editing trumps review for the tips; but policy related material needs to be on the web site.